src/App.tsx

```diff
@@ -1,61 +1,109 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [userSelect, setUserSelect] = useState('0');
+  const [title, setTitile] = useState('');
+  const [todos, setTodos] = useState(todosFromServer);
+  const [submit, setSubmit] = useState(true);
+
+  const trimedTitle = title.trim();
+
+  const largestTodoId = todos.reduce((previously, currently) => (
+    previously.id > currently.id ? previously : currently
+  ));
+
+  const updateTodoList = () => {
+    const newTodo = {
+      id: largestTodoId.id + 1,
+      title: trimedTitle,
+      completed: false,
+      userId: Number(userSelect),
+    };
+
+    setTodos(old => (
+      [...old, newTodo]
+    ));
+    setTitile('');
+    setUserSelect('0');
+    setSubmit(true);
+  };
+
+  const handleSubmit = (event:React.MouseEvent) => {
+    setSubmit(false);
+    event.preventDefault();
+    if (trimedTitle !== '' && userSelect !== '0') {
+      setSubmit(true);
+
+      return updateTodoList();
+    }
+
+    setSubmit(false);
+
+    return null;
+  };
+
   return (
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/users" method="POST">
+      <form>
         <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
+          <label htmlFor="title">
+            Title:
+          </label>
+          <input
+            id="title"
+            type="text"
+            data-cy="titleInput"
+            placeholder="Enter a title"
+            value={title}
+            onChange={event => setTitile(event.target.value)}
+          />
+          {
+            !trimedTitle && !submit
+              ? <span className="error">Please enter a title</span>
+              : null
```

Comment:
```suggestion
            !trimedTitle && !submit && (
              <span className="error">Please enter a title</span>
            )
```
You can do same without ternary operator